src/scopes.ts

```diff
@@ -0,0 +1,11 @@
+export const SCOPES = [
```

Comment:
This might make it less flexible in the sense that if a new scope is created on the API server, we will have to release a new SDK.